### PR TITLE
added flex layout with button styling for components of the status page

### PR DESCRIPTION
**Pull request by melbgirl**
Very simple update to the status page to add flex layout, and leveraging the button styling. This is only one part of the changes needed for[1570](https://github.com/open-telemetry/opentelemetry.io/issues/1570)

### CODE REVIEW

File: content/en/status.md [hunk 1]
```diff
@@ -12,11 +12,31 @@ description: Maturity-level of the main OpenTelemetry components
...
+- [.NET](/docs/instrumentation/net/) 
+- [Erlang/Elixir](/docs/instrumentation/erlang/) 
+- [Go](/docs/instrumentation/go/) 
+- [Java](/docs/instrumentation/java/) 
+- [JavaScript](/docs/instrumentation/js/) 
+- [PHP](/docs/instrumentation/php/) 
+- [Python](/docs/instrumentation/python/) 
+- [Ruby](/docs/instrumentation/ruby/) 
+- [Rust](/docs/instrumentation/rust/) 
+- [Swift](/docs/instrumentation/swift/)
```
Comment (chalin) [inline content/en/status.md:36]: Add the anchors to the links: `#status-and-releases`

**Review by chalin — COMMENTED** [review]
Nice step forward. See inline comment

**Comment by chalin** [comment]
Hi @melbgirl: when do you expect to be able to make the updates to this PR? Would you like some help?

**Comment by melbgirl** [comment]
I’m planning to do some work on this this coming weekend. LMK if that’s too far out.

**Review by chalin — APPROVED** [review]
Thanks!

**Comment by chalin** [comment]
Going to merge this since it has the necessary updates we had agreed to during the last Comms meeting.

**Pull request merged**